lib/preload.js

```diff
@@ -1075,11 +1076,13 @@ class Preloader extends EventEmitter {
 				// Docker connection
 				// We use localhost on windows because of this bug in node < 8.10.0:
 				// https://github.com/nodejs/node/issues/14900
-				const innerDockerProgress = new dockerProgress.DockerProgress({
-					Promise: Bluebird,
+				const innerDocker = new Docker({
 					host: os.platform() === 'win32' ? 'localhost' : '0.0.0.0',
 					port: this.dockerPort,
 				});
+				const innerDockerProgress = new dockerProgress.DockerProgress({
+					docker: innerDocker,
```

Comment:
This is done to use the new interface of `docker-progress` v5:
https://github.com/balena-io-modules/docker-progress/pull/54

It allows `docker-progress` to use the same version (and even the same instance) of `dockerode` as `balena-preload` and `balena-cli`.